protos/protos.go

```diff
@@ -179,7 +179,7 @@ func (protocols ProtocolsStruct) GetAllUdp() map[Protocol]UdpProtocolPlugin {
 // will match against packets for the registered protocols. If with_vlans is
 // true the filter will match against both IEEE 802.1Q VLAN encapsulated
 // and unencapsulated packets
-func (protocols ProtocolsStruct) BpfFilter(with_vlans bool) string {
+func (protocols ProtocolsStruct) BpfFilter(with_vlans bool, with_icmp bool) string {
```

Comment:
at some point I would prefer a bitmask or a struct of booleans over multiple boolean parameters.